ledger/complete/ledger.go

```diff
@@ -409,17 +411,18 @@ func (l *Ledger) ExportCheckpointAt(
 
 	l.logger.Info().Msgf("checkpoint file successfully stored at: %v %v", outputDir, outputFile)
 
-	l.logger.Info().Msgf("generating reports")
+	l.logger.Info().Msgf("finished running post-checkpoint reporters")
 
-	// run reporters
-	for _, reporter := range reporters {
-		err := runReport(reporter, payloads, statecommitment, l.logger)
+	// running post checkpoint reporters
+	for i, reporter := range postCheckpointReporters {
+		l.logger.Info().Msgf("running a post-checkpoint generation reporter: %s, (%v/%v)", reporter.Name(), (i, len(postCheckpointReporters))
```

Comment:
```suggestion
		l.logger.Info().Msgf("running a post-checkpoint generation reporter: %s, (%v/%v)", reporter.Name(), i, len(postCheckpointReporters))
```

This is causing build errors.